app-localize-behavior.html

```diff
@@ -215,6 +215,9 @@
     var proto = this.constructor.prototype;
 
     // Everytime any of the parameters change, invalidate the strings cache.
+    if (!proto.__localizationCache) {
+      proto.__localizationCache = {};
```

Comment:
This doesn't need to fill all the properties of `__localizationCache` like [this line](https://github.com/PolymerElements/app-localize-behavior/pull/54/files#diff-eb3c5a91b266417d17c72bdbbdf0f0e8R184)?